EsFormatter.py

```diff
@@ -119,10 +119,10 @@ def run(self, edit, save=False, ignoreSelection=False):
 
     def replaceFile(self, thread, save=False):
         '''Replace the entire file content with the formatted text.'''
+        sublime.status_message("File formatted")
```

Comment:
I agree with the other change, but why changing this? It the file didn't change, there's no need to show the message. What do you think?